lib/generators/boring/devise/jwt/install/install_generator.rb

```diff
@@ -0,0 +1,219 @@
+        end
+
+        def configure_denylist_strategy
+          Bundler.with_unbundled_env do
+            run "bundle exec rails generate model jwt_denylist --skip-migration"
+            run "bundle exec rails generate migration create_jwt_denylist"
+          end
+
+          migration_content = <<~RUBY
+            t.string :jti, null: false
+            t.datetime :exp, null: false
+          RUBY
+
+          gsub_file Dir["db/migrate/*_create_jwt_denylist.rb"][0],
+                    /create_table :jwt_denylists do \|t\|/,
+                    "create_table :jwt_denylist do |t|",
+                    verbose: false
+
+          inject_into_file Dir["db/migrate/*_create_jwt_denylist.rb"][0],
+                           optimize_indentation(migration_content, 6),
+                           after: /create_table :jwt_denylist do \|t\|\n/,
+                           verbose: false
+
+          inject_into_file Dir["db/migrate/*_create_jwt_denylist.rb"][0],
+                           optimize_indentation("add_index :jwt_denylist, :jti", 4),
+                           before: /^  end/,
+                           verbose: false
+
+          add_devise_jwt_module("JwtDenylist")
+          
+          jwt_denylist_content = <<~RUBY
+            include Devise::JWT::RevocationStrategies::Denylist
+            self.table_name = 'jwt_denylist'
+          RUBY
+
+          inject_into_file "app/models/jwt_denylist.rb",
+                            optimize_indentation(jwt_denylist_content, 2),
+                            after: /ApplicationRecord\n/,
+                            verbose: false
+        end
+
+        def configure_allowlist_strategy
+          model_name = options[:model_name].underscore
+          Bundler.with_unbundled_env do
+            run "bundle exec rails generate model allowlisted_jwt"
+          end
+
+          migration_content = <<~RUBY
+            t.string :jti, null: false
+            t.string :aud
+            # If you want to leverage the `aud` claim, add to it a `NOT NULL` constraint:
+            # t.string :aud, null: false
+            t.datetime :exp, null: false
+            t.references :#{model_name}, foreign_key: { on_delete: :cascade }, null: false
+          RUBY
+
+          inject_into_file Dir["db/migrate/*_create_allowlisted_jwts.rb"][0],
+                           optimize_indentation(migration_content, 6),
+                           after: /create_table :allowlisted_jwts do \|t\|\n/,
+                           verbose: false
+
+          inject_into_file Dir["db/migrate/*_create_allowlisted_jwts.rb"][0],
+                           optimize_indentation("add_index :allowlisted_jwts, :jti", 4),
+                           before: /^  end/,
+                           verbose: false
+
+          add_devise_jwt_module("self", include_content: "include Devise::JWT::RevocationStrategies::Allowlist")
+        end
+
+        def add_devise_jwt_module(strategy, include_content: nil)
```

Comment:
Let's update this method to take named arguments so when calling this method it will be more clear on what's happening e.g.  add_devise_jwt_module strategy: "...", include_content: "..."